spock-specs/src/test/groovy/org/spockframework/smoke/parameterization/UnrolledFeatureMethods.groovy

```diff
@@ -461,3 +494,26 @@ def "an actor (named #actor.getName()) age #actor.getAge()"() {
     }
   }
 }
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target([ElementType.TYPE])
+@Inherited
```

Comment:
This is just superfluous here, isn't it?

Reply:
yes it was c&p from the example code.

Reply:
So, maybe it could be removed?